tools/dependencies-report/src/main/resources/notices/org.eclipse.core.contenttype-3.4.100-NOTICE.txt

```diff
@@ -0,0 +1,88 @@
+Copyright © Eclipse Foundation, Inc. All Rights Reserved.
```

Comment:
the same is true for the rest of the org.eclipse packages; because Eclipse Foundation assumes ownership and maintenance of projects that were bootstrapped by corporations like IBM, it's important to include the _original_ copyright owners here.